emulators/libkrun/files/patch-makefile.diff

```diff
@@ -0,0 +1,26 @@
+--- Makefile	2024-09-27 19:06:27
++++ Makefile.new	2024-12-27 01:31:54
+@@ -71,7 +71,7 @@
+ LIBDIR_Darwin = lib
+ 
+ ifeq ($(PREFIX),)
+-    PREFIX := /usr/local
++    PREFIX := @@DEFAULT_PREFIX@@
+ endif
```

Comment:
This seems to be setup so that we could just pass `PREFIX` to make on the command line, and that would avoid the reinplace step? The other hunk of this patch would still be required, though, so this is more of a nitpick than anything else.

Reply:
Could try it that way, would you prefer it with `PREFIX`? I only went the current route by grepping this repo for existing examples of correcting prefixes in Makefiles and saw that this pattern is used elsewhere often.

I'm not married to either.

Reply:
I'm not married to either solution either. This works, and you already have it implemented, so let's just leave it as-is.